### ISSUE TITLE
Incorrect path?

### ISSUE THREAD

**Issue by melvn** [smartbugs/smartbugs#140]
Tools and analysis run sucessfully but the results file outputs the same error of, regardless of what tool is chosen

> "exec /sb/bin/do_solidity: no such file or directory"

Looking for the do_solidity file i found it the tools path of each analyser respectively. Possible that bin should be redirected towards tools sub directory? No error is thrown during analysis.

**Comment by gsalzer** [comment]
The `scripts` folders in the `tools`directory are mounted inside the docker image, as `/sb/bin`. So the error indicates that something went wrong with mounting the folder, as the script `do_solidity` cannot be found. We will try to reproduce the problem and solve it.

**Comment by gsalzer** [comment]
Memo: this issue was reported for Windows (it is a follow-up on issue #139)

**Comment by jff** [comment]
Hi @melvn , could you provide more details about your platform? In particular:

- SmartBugs version: [e.g. 2.0.2]
- Python version: [e.g. 3.10.6]
- OS : [e.g. Linux, Ubuntu, MacOS, Windows, ...]
- CPU: [e.g. Intel, AMD, M1, ... ]

**Comment by gsalzer** [comment]
@melvn: to obtain the infos asked for by @jff, it should suffice to do `python -m sb --version`, after pulling the most recent version of SB and activating the virtual environment.

**Comment by melvn** [comment]
Smartbugs Version: 2.0.1
Python Version: 3.10.9
OS: Windows 10 Home 10.0.19444
CPU: Intel(R) Core i5-7400

**Comment by lmfuerst** [comment]
I had the same problem when using Win 10; manually updating the shell scripts (e.g. tools/conkas/scripts) from CRLF to LF seems to have fixed it.

**Comment by gsalzer** [comment]
@lmfuerst Thanks, this is helpful to solve the problem.

**Comment by gsalzer** [comment]
@lmfuerst Can you please check again with the latest commit of SmartBugs? I have added a file `.gitattributes` in the home dir that is supposed to tell git to use just LF as line endings for *.sh files.
The easiest way is probably to do a fresh clone. Otherwise, you might try to delete just the `tools` folder and then do `git reset --hard HEAD`. If everything works as expected, the scripts should run without having to edit them.

**Comment by lmfuerst** [comment]
I've tried a fresh clone, however the analysis seems to get stuck now/doesn't execute. The last output I'm seeing when running e.g. `python -m sb -t mythril -f file.sol` is `Starting task 1/1: mythril-0.23.15 and file.sol`, after that nothing happens even after waiting some time. Tried the same yesterday, where it completed within a couple of seconds.

The shell files `do_solidity`, `do_runtime` and `do_bytecode` are however missing in the fresh clone; were they removed on purpose?
